bin/gwdetchar-scattering

```diff
@@ -124,6 +124,8 @@ os.chdir(args.output_dir)
 
 segfile = '%s-SCATTERING_SEGMENTS_%s_HZ-%s.h5' % (args.ifo, tstr, gpsstr)
 
```

Comment:
@siddharth101, can you remove this empty line, so that `segfile` and `summfile` are grouped together contextually?

Reply:
Done.